docs/content/Template.mdx

```diff
@@ -0,0 +1,313 @@
+  onAction={item => console.log(item)}
+  overlayProps={{width: 'medium'}}
+  items={[
+    {
+      text: 'primer/reviewers',
+      description: '4 members',
+      descriptionVariant: 'block',
+      leadingVisual: () => <Avatar src="https://github.com/primer.png" />,
+      key: 'primer/reviewers'
+    },
+    ActionMenu.Divider,
+    {
+      text: 'langermank',
+      description: 'Katie Langerman',
+      descriptionVariant: 'inline',
+      leadingVisual: () => <Avatar src="https://github.com/langermank.png" />,
+      key: 'langermank'
+    },
+    {
+      text: 'maximedegreve',
+      description: 'Maxime De Greve',
+      descriptionVariant: 'inline',
+      leadingVisual: () => <Avatar src="https://github.com/maximedegreve.png" />,
+      key: 'maximedegreve'
+    },
+    {
+      text: 'alliethu',
+      description: 'Allie Thu',
+      descriptionVariant: 'inline',
+      leadingVisual: () => <Avatar src="https://github.com/alliethu.png" />,
+      key: 'alliethu'
+    }
+  ]}
+/>
+```
+
+### Example with groups
+
+```jsx live
+<ActionMenu
+  anchorContent="Menu"
+  onAction={item => console.log(item)}
+  groupMetadata={[
+    {groupId: '0'},
+    {groupId: '1', header: {title: 'Live query', variant: 'subtle'}},
+    {groupId: '2', header: {title: 'Layout', variant: 'subtle'}}
+  ]}
+  items={[
+    {key: '1', leadingVisual: TypographyIcon, text: 'Rename', groupId: '0'},
+    {key: '2', leadingVisual: VersionsIcon, text: 'Duplicate', groupId: '0'},
+    {key: '3', leadingVisual: SearchIcon, text: 'repo:github/github', groupId: '1'},
+    {
+      key: '4',
+      leadingVisual: NoteIcon,
+      text: 'Table',
+      description: 'Information-dense table optimized for operations across teams',
+      descriptionVariant: 'block',
+      groupId: '2'
+    },
+    {
+      key: '5',
+      leadingVisual: ProjectIcon,
+      text: 'Board',
+      description: 'Kanban-style board focused on visual states',
+      descriptionVariant: 'block',
+      groupId: '2'
+    }
+  ]}
+/>
+```
+
+### Example with Custom Anchor
+
+import {Box} from '@primer/components'
+
+```jsx live
+<Box bg="canvas.subtle" p={2}>
+  <Label outline mr={2}>
+    Contributor
+  </Label>
+  <SmileyIcon color="fg.muted" />
+  <ActionMenu
+    renderAnchor={anchorProps => (
+      <ButtonInvisible {...anchorProps} color="fg.subtle">
+        <KebabHorizontalIcon />
+      </ButtonInvisible>
+    )}
+    onAction={item => console.log(item)}
+    items={[
+      {text: 'Copy link', key: 'copy'},
+      {text: 'Quote reply', key: 'quote'},
+      ActionMenu.Divider,
+      {text: 'Edit', key: 'edit'},
+      ActionMenu.Divider,
+      {text: 'Report content', key: 'report'}
+    ]}
+  />
+</Box>
+```
+
+<br />
+
+## API Reference
+
+<!-- collocate imports -->
+
+import {Props} from '../src/props'
+import {ActionMenu} from '@primer/components'
+
+<!-- List root component and it's children -->
+
+<Props of={ActionMenu} />
+
+<br />
+
+## Best practices
```

Comment:
These best practices don't seem implementation-specific. I wonder if we should just link to the corresponding documentation in our [interface guidelines](https://primer.style/design/components/action-list) to avoid duplicating content.

Reply:
I'm split here. On one hand, yes absolutely! On the other, I don't know if folks will read the other page before copying examples :(